src/ParkDetails/ParkDetails.js

```diff
@@ -6,31 +6,73 @@ function ParkDetails({ foundPark }) {
   if (!foundPark) {
     return null;
   }
-  console.log(foundPark);
+
+  let parkImage;
+  if (foundPark.attributes.media && foundPark.attributes.media.length > 1) {
+    parkImage = (
+      <img
+        className="single-image"
+        src={foundPark.attributes.media[1].url}
+        alt={foundPark.attributes.media[1].altText}
```

Comment:
interesting use of the let variable to allow for the smoother rendering, im glad you were able to fix this bug!